test/runtime/fqdn.go

```diff
@@ -331,6 +332,64 @@ var _ = Describe("RuntimeFQDNPolicies", func() {
 		res.ExpectSuccess("Cannot access to allowed DNS name %q", world1Target)
 	})
 
+	It("Validate dns-proxy monitor information", func() {
+		policy := `
+[
+	{
+		"labels": [{
+			"key": "monitor"
+		}],
+		"endpointSelector": {
+			"matchLabels": {
+				"container:id.app1": ""
+			}
+		},
+		"egress": [
+			{
+				"toPorts": [{
+					"ports":[{"port": "53", "protocol": "ANY"}],
+					"rules": {
+						"dns": [
+							{"matchPattern": "world1.cilium.test"}
+						]
+					}
+				}]
+			},
+			{
+				"toFQDNs": [{
+					"matchPattern": "world1.cilium.test"
+				}]
+			}
+		]
+	}
+]`
+		_, err := vm.PolicyRenderAndImport(policy)
+		Expect(err).To(BeNil(), "Policy cannot be imported")
+
+		expectFQDNSareApplied()
+
+		ctx, cancel := context.WithCancel(context.Background())
+		monitorCMD := vm.ExecInBackground(ctx, "cilium monitor")
```

Comment:
Let's add `--type l7` here, since the DNS stuff is layer 7 and we then implicitly test that we emit it at the correct layer.